pylabrobot/tilting/hamilton_backend.py

```diff
@@ -26,6 +27,7 @@ async def setup(self):
       write_timeout=self.write_timeout,
       timeout=self.timeout)
 
+    await self.tilt_initial_offset(initial_offset)
     await self.send_command("SI")
```

Comment:
does tilting before sending SI work?

Reply:
after a power cycle

Reply:
No, even when I initialize `self.ser`, the tilt module returns error code `3: "Not initialized"`.

Reply:
so that means command SI needs to be sent first

Reply:
As far as I can tell, yes.